packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/configuration/onPremise/ovhPcc/dedicatedCloud-datacenter-drp-onPremise-ovhPccStep.routing.js

```diff
@@ -62,7 +59,7 @@ export default /* @ngInject */ ($stateProvider) => {
       {
         url: '/legacy-order-ip',
         controller: 'IpLegacyOrderCtrl',
-        template: legacyTemplate,
+        template: 'ip/legacyOrder/ip-ip-legacyOrder.html',
```

Comment:
why not 
```
import template from './ip-ip-legacyOrder.html';
```
and avoid put it in $templateCache?

Reply:
because it would be something like this
```js
import template from '../../../../../ip/etc';
```
but i realise that using template cache string here is a mistake since ip module might not be loaded yet
i'm going to fix it